entries/w/woltlab.com.json

```diff
@@ -1,15 +1,16 @@
 {
   "WoltLab": {
     "domain": "woltlab.com",
+    "tfa": [
+      "totp",
+      "u2f",
+      "email"
+    ],
+    "documentation": "https://www.woltlab.com/account-security/",
     "url": "https://www.woltlab.com/",
     "img": "woltlab.com.png",
```

Comment:
Since you're working for WoltLab, you wouldn't happen to have the logo in an SVG format? We try to use SVG icons for all entries but we haven't been able to find one for WoltLab thus the PNG at `img/w/woltlab.com.png`.

Reply:
We have an SVG available, but due to the large number of colors in our logo it is much larger than the current PNG:

- The raw SVG is 7472 Bytes.
- The optimized SVG coming out of svgo is 6076 Bytes.
- The current PNG is exactly 1024 Bytes.

I can add the SVG if you like, but I'm not sure this is the intended outcome?

Reply:
I wouldn't mind a 6KB SVG icon. There are several ones that are bigger already, and it will automatically be zipped when serving the site as well (I hope), so it's not a big difference to an already compressed PNG of size 1KB.

Reply:
@phallobst Okay, I've just updated the image in a separate commit. `gzip -1` for the SVG results in 2280 Bytes for me. I'm happy to drop the commit again if you don't feel like the SVG is an improvement. Thanks!